src/utils.js

```diff
@@ -38,7 +38,11 @@ export function standardizeWitUnits(amount, unit) {
     [`${WIT_UNIT.MICRO}`]: 3,
     [`${WIT_UNIT.NANO}`]: 0,
   }
-  return (amount / Math.pow(10, units[unit])).toFixed(units[unit]).replace(/\.?0+$/, '')
+  if (unit === WIT_UNIT.NANO) {
+    return amount
```

Comment:
It looks a little strange to me that one branch here returns `Number` while the other returns `String`.

Most likely it's not going to be a problem in our case, but I'd sleep better at night with this:

```suggestion
    return amount.toString()
```

Sorry for the nitpicking :sweat_smile: